app/javascript/flavours/glitch/reducers/meta.js

```diff
@@ -13,14 +13,10 @@ const initialState = ImmutableMap({
 export default function meta(state = initialState, action) {
   switch(action.type) {
   case STORE_HYDRATE:
-    return state.merge(
-      action.state.get('meta'))
-        .set('permissions', action.state.getIn(['role', 'permissions']))
-        .set('layout', layoutFromWindow(action.state.getIn(['local_settings', 'layout']))
-      );
+    return state.merge(action.state.get('meta')).set('permissions', action.state.getIn(['role', 'permissions'])).set('layout', layoutFromWindow(action.state.getIn(['local_settings', 'layout'])));
```

Comment:
I see that this abides to style rules, but I don't particularly find this easy to read.
Upon further inspection, I don't like this return at all.
I initially thought about changing the indent rule, but it wouldn't help in this case at all.
The `);` confused me.

Maybe something like this, which both maintains readability while abiding to style rules:

```suggestion
    return state.merge(action.state.get('meta'))
      .set('permissions', action.state.getIn(['role', 'permissions']))
      .set('layout', layoutFromWindow(action.state.getIn(['local_settings', 'layout'])))
    ;
```

I don't particularly like the standalone semi there either, but it communicates the return statement is ending there. (Alternatively the semi can also be set after the second .set)